velox/functions/prestosql/tests/BinaryFunctionsTest.cpp

```diff
@@ -642,4 +642,23 @@ TEST_F(BinaryFunctionsTest, toIEEE754Bits64) {
       toIEEE754Bits64(std::numeric_limits<double>::lowest()));
 }
 
+TEST_F(BinaryFunctionsTest, fromIEEE754Bits64) {
+  const auto fromIEEE754Bits64 = [&](const std::optional<std::string>& arg) {
+    return evaluateOnce<double>("from_ieee754_64(from_hex(c0))", arg);
+  };
+  EXPECT_EQ(std::nullopt, fromIEEE754Bits64(std::nullopt));
+  EXPECT_EQ(1, fromIEEE754Bits64("3FF0000000000000"));
```

Comment:
Can you add a couple of tests for `fromIEEE754Bits64(toIEEE754Bits64())`.

Reply:
> Can you add a couple of tests for `fromIEEE754Bits64(toIEEE754Bits64())`?

I could add some tests for `fromIEEE754Bits64(toIEEE754Bits64())` but  I am currently correcting the semantics of toIEEE754Bits64() to return 64 bits instead of 64 bytes so as per my knowledge that needs to be merged first?

Reply:
Yes, that needs to be fixed first.